src/lib/components/backoffice/ExportSearchResults/ExportSearchResults.js

```diff
@@ -105,6 +105,7 @@ class ExportSearchResultsWithState extends Component {
 
     // append the `format` param
     const params = Qs.parse(queryString);
+    params['page'] = 1; // Start from the first page
```

Comment:
how will it behave with other pages?

Reply:
The button has the same effect, regardless the currently selected page. Namely it exports the results of the search without taking pagination into account. I assumed this was the intended semantic of that button since it's said to return the "first 10,000 records".
Did I miss something?

Reply:
I need to investigate this further to make sure, I will merge this and start a ticket